Makefile

```diff
@@ -24,7 +24,7 @@ TESTBIN := build/bin/${BIN}-${GOOS}-${GOARCH}
 all: build
 
 prebuild:
-	go get -u github.com/gobuffalo/packr/packr
+	go install github.com/gobuffalo/packr/packr
 	packr
```

Comment:
This one was a mystery why is it being run after being installed as there was no obvious side-effect in the project own files/directories; thought maybe it's just used to verify `packr` tool was installed correctly, that it can be run, but there was no other place in `Makefile` or in CircleCI build definition where it's being used.

It turned out this `packr` run without any additional parameters/commands is actually a must, part of [using packr "the hard way"](https://github.com/gobuffalo/packr#building-a-binary-the-hard-way), which as a result generates `-packr.go` files for all `packr` `Box`es for a project, all of which for `gsctl` (and `opsctl`) are defined in the shared `gscliauth` library, and since `gsctl` dependencies are managed with Go modules, `-packr.go` files actually end up in go modules cache, under `$GOPATH/pkg/mod` (or `$HOME/go/pkg/mod` if `GOPATH` is not set), so outside of the project directory. Not sure would it make sense to add a comment above or on the same line with `packr` binary usage with reference to ["the hard way"](https://github.com/gobuffalo/packr#building-a-binary-the-hard-way), maybe it's obvious to others, until now `packr` wasn't in my toolbox.